### PR TITLE
AWS Lambda : CDS-52148

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/onboard-cd/cd-quickstarts/aws_lambda.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Harness supports the deployment of AWS Lambda Functions. Below is a guide to get
 
 *Note: AWS native lambda feature is behind FF: `CDS_AWS_NATIVE_LAMBDA`*
 
+## Product Demo Video
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/onboard-cd/cd-quickstarts/aws_lambda.md:9]:
```suggestion
## Visual summary
```

**Review by roshnisarangadharan — DISMISSED** [review]

**Review by roshnisarangadharan — APPROVED** [review]

**Comment by michaelcretzman** [comment]
@roshnisarangadharan @thisrohangupta this PR is broken. It can't be built without errors.
Let me look into it and integrate the content later this week.

**Review by michaelcretzman — CHANGES_REQUESTED** [review]
@roshnisarangadharan @thisrohangupta this PR is broken. It can't be built without errors.
Let me look into it and integrate the content later this week.

**Comment by michaelcretzman** [comment]
Duplicate of https://github.com/harness/developer-hub/pull/1362

**Pull request closed without merging**